apps/docs/src/content/components/accordion.mdx

```diff
@@ -1,78 +1,8 @@
-import {
-  accordionClasses as classes,
-  HvAccordion,
-} from "@hitachivantara/uikit-react-core";
-
-import { Header } from "../../components/Header";
-
-<Header
-  params={{ name: "Accordion", classes }}
-  Component={HvAccordion}
-  componentName={"HvAccordion"}
-  controls={{
-    label: { type: "text", defaultValue: "Accordion label" },
-    labelVariant: { defaultValue: "label" },
-    disabled: { defaultValue: false },
-  }}
-  componentProps={{ style: { width: "100%" } }}
->
-  <div>My accordion example content</div>
-</Header>
-
-> [!NOTE]
+> [!IMPORTANT]
 >
-> You can use the `HvSection` component to create an accordion group that better
-> fits your needs. Please refer to the [Section
+> You should use the `HvSection` component to create accordions. Please refer to the [Section
 > documentation](/components/section#multiple) for usage.
 
-### Controlled
-
-You can control the state of `HvAccordion` by using the `expanded`/`onChange` props.
-
-```tsx live
-import { useState } from "react";
```

Comment:
We talked about this. We own the `/components/*`, which is where we document the technical documentation of our components. `HvAccordion` is one of our top-level components which we currently offer, therefore it should be documented.